src/components/ebay-progress-spinner/index.marko

```diff
@@ -0,0 +1,2 @@
+$ var label = input.a11yText ? input.a11yText : "Busy"
+<span ...input class=[ "spinner", input.size === "large" && "spinner--large"] aria-label=label role="img"></span>
```

Comment:
Just so it's here (we chatted about this), use `processHTMLAttributes` instead and set `a11yText` and `size` in the ignored attributes.